src/main/drivers/barometer/barometer_bmp280.c

```diff
@@ -125,7 +126,9 @@ bool bmp280Detect(baroDev_t *baro)
     }
 
     // read calibration
-    busReadRegisterBuffer(busdev, BMP280_TEMPERATURE_CALIB_DIG_T1_LSB_REG, (uint8_t *)&bmp280_cal, 24);
+    STATIC_ASSERT(sizeof(bmp280_calib_param_t) == BMP280_PRESSURE_TEMPERATURE_CALIB_DATA_LENGTH, bmp280_calibration_structure_incorrectly_packed);
```

Comment:
Can we have this close to the definition of `bmp280_calib_param_t` for clarity?

Reply:
sure, i'll update that commit.

Reply:
You seem to have missed this.

Reply:
that's odd, i made the commit, maybe i forgot to push the branch... 1 sec.